HPXMLtoOpenStudio/measure.rb

```diff
@@ -4245,11 +4246,41 @@ def self.set_surface_exterior(model, spaces, surface, surface_id, exterior_adjac
     elsif ['basement - conditioned'].include? exterior_adjacent_to
       surface.createAdjacentSurface(create_or_get_space(model, spaces, 'living space'))
       @cond_bsmnt_surfaces << surface
+    elsif ['other heated space', 'other multifamily buffer space', 'other non-freezing space'].include? exterior_adjacent_to
+      add_otherside_coefficients(surface, exterior_adjacent_to, model, spaces)
     else
       surface.createAdjacentSurface(create_or_get_space(model, spaces, exterior_adjacent_to))
     end
   end
 
+  def self.add_otherside_coefficients(surface, exterior_adjacent_to, model, spaces)
+    if spaces[exterior_adjacent_to].nil?
+      otherside_object = OpenStudio::Model::SurfacePropertyOtherSideCoefficients.new(model)
+      if exterior_adjacent_to == 'other heated space'
+        temp_min = UnitConversions.convert(68, "F", "C")
+        indoor_weight = 0.5
+        outdoor_weight = 0.5
+      elsif exterior_adjacent_to == 'other multifamily buffer space'
+        temp_min = UnitConversions.convert(50, "F", "C")
+        indoor_weight = 0.5
+        outdoor_weight = 0.5
+      elsif exterior_adjacent_to == 'other non-freezing space'
+        temp_min = UnitConversions.convert(40, "F", "C")
+        indoor_weight = 0.0
+        outdoor_weight = 1.0
+      end
+      otherside_object.setName(exterior_adjacent_to)
+      otherside_object.setCombinedConvectiveRadiativeFilmCoefficient(30)
+      otherside_object.setMinimumOtherSideTemperatureLimit(temp_min)
+      otherside_object.setZoneAirTemperatureCoefficient(indoor_weight)
+      otherside_object.setExternalDryBulbTemperatureCoefficient(outdoor_weight)
+      surface.setSurfacePropertyOtherSideCoefficients(otherside_object)
+      spaces[exterior_adjacent_to] = otherside_object
+    else
+      surface.setSurfacePropertyOtherSideCoefficients(spaces[exterior_adjacent_to])
```

Comment:
I'm letting the surfaces adjacent to the same new space type share one single `SurfacePropertyOtherSideCoefficients` object, for surfaces that are not adjacent to the same interior space, this will cause issues. Will switch to one object per surface.